$MirOS: ports/games/bsdgames/patches/patch-fortune_datfiles_Makefile,v 1.1.7.1 2005/03/18 15:46:07 tg Exp $
--- fortune/datfiles/Makefile.orig	Thu Jan  1 00:00:00 1970
+++ fortune/datfiles/Makefile	Thu Mar  2 01:38:10 2006
@@ -3,20 +3,26 @@
 #	@(#)Makefile	8.2 (Berkeley) 4/19/94
 
 SRCS=	fortunes fortunes2 startrek zippy recipes
-BLDS=	fortunes.dat fortunes2.dat startrek.dat zippy.dat \
-	fortunes-o fortunes-o.dat recipes.dat
+BLDS=
 
-# TO INSTALL THE POTENTIALLY OFFENSIVE FORTUNES, UNCOMMENT THE THREE
-# LINES AND COMMENT OUT THE FOURTH LINE.
-
-# THE THREE LINES
-SRCS+=	fortunes2-o limerick
-BLDS+=	fortunes2-o.dat limerick.dat
+SRCS+=	fortunes-o fortunes2-o limerick
 TYPE=	real
 
 # THE FOURTH LINE
 #TYPE=	fake
 
+# fortune-mod-fvl 20030120
+SRCS+=	at.linux fvl fvl_vendor martin.piskernig
+SRCS+=	norbert.tretkowski rss thomas.ogrisegg
+# fortune-fh-brs 20060301
+SRCS+=	fh-brs
+
+#----
+
+.for _i in ${SRCS}
+BLDS+=	${_i}.dat
+.endfor
+
 CLEANFILES+=${BLDS}
 
 STRFILE!=cd $(.CURDIR)/../strfile; \
@@ -26,18 +32,15 @@ STRFILE!=cd $(.CURDIR)/../strfile; \
 all: ${BLDS}
 
 install: ${SRCS} ${BLDS}
-	(cd ${.CURDIR} && ${INSTALL} ${INSTALL_COPY} -o ${BINOWN} -g ${BINGRP} -m 444 \
-	    ${SRCS} ${DESTDIR}/usr/share/games/fortune)
-	${INSTALL} ${INSTALL_COPY} -o ${BINOWN} -g ${BINGRP} -m 444 ${BLDS} \
-	    ${DESTDIR}/usr/share/games/fortune
+	${INSTALL} ${INSTALL_COPY} -o ${BINOWN} -g ${BINGRP} -m ${SHAREMODE} \
+	    $> ${DESTDIR}/usr/share/games/fortune/
 
-fortunes.dat fortunes2.dat fortunes2-o.dat limerick.dat startrek.dat zippy.dat recipes.dat:
-	${STRFILE} -rs ${.CURDIR}/${.TARGET:R} ${.TARGET}
+.for _i in ${SRCS}
+${_i}.dat: ${_i}
+	${STRFILE} -rs $> $@
+.endfor
 
-fortunes-o.dat: fortunes-o
-	${STRFILE} -rsx fortunes-o ${.TARGET}
-
 fortunes-o: fortunes-o.${TYPE}
-	tr [a-zA-Z] [n-za-mN-ZA-M] < ${.ALLSRC} > ${.TARGET}
+	cp $> $@
 
 .include <bsd.prog.mk>
